### ISSUE TITLE
I followed the video that was made but..

### ISSUE THREAD

**Issue by nissan24zero** [TormStorm/jagex-launcher-linux#30]
I got as far as having the launcher in steam OS.
but when I go to click play for an rs3 character,  nothing happens.

**Comment by TormStorm** [comment]
Do you have the flatpak RuneScape client installed on your system?
https://flathub.org/apps/com.jagex.RuneScape

**Comment by nmlynch94** [comment]
Yes this is likely why. It should be available in the software center on the steam deck.

**Comment by nissan24zero** [comment]
I'll look into that when i get off work tonight. 
Thanks for the video by the way,  extremely helpful.  Really held my hand through the process lol

**Comment by nissan24zero** [comment]
So I had rs3 installed, but it didn't play the game at all. 
I installed osrs directly from the launcher, and it loaded up fine but the resolution was wrong,  i was going to deal with that later,  but today I tried again and the launcher wouldn't even open. 
So I Uninstalled the launcher from lutris, removed the launcher from my non steam games and tried to start over from scratch,  but now lutris won't even open the script file folder. 
I even tried reinstalling lutris but no success.

**Comment by nmlynch94** [comment]
Hmm I’m not sure why the flatpak launch wasn’t working…I’ll test that again when I get back to my deck to make sure everything is working.

as for the file selector not opening, this is a bug on the steam deck. I’m mobile so don’t have the link handy but checkout the troubleshooting steps in the readme for a systemctl command to fix it.

you might have to run it in desktop mode first, then try to get the settings tweaked right.

I had assumed at the beginning of this that you wanted to play OSRS. Since you’re playing rs3, have you tried the steam version?

**Comment by nissan24zero** [comment]
So I got lutris to work again from the advice you gave in the readme file. 
Reinstalled the launcher successfully. 
I tried using the steam version and flatpak version of runescape and neither load in steam OS after hitting play. 
I was able to successfully get runescape 3 to play in desktop mode with the flatpak, haven't tried steam version on desktop mode.  But desktop mode defeats the purpose I'm trying to achieve due to not  having the button remap capabilities.  
Am I missing something?
All I did was installed the launcher,  then installed the game.
While I'm here, this might be a silly question,  but is it possible to have alt1 on deck as well through this installation process?
I know I'm getting ahead of myself,  but if it's possible, I have faith in you guys directing me in the way I want to go.

**Comment by nmlynch94** [comment]
Nice!

Did you swap the version of proton you were using in runescape? I downloaded and was able to launch it in Game mode with no modifications. May be worth trying a reinstall or reviewing the protondb page so see if any of those specific workarounds work for you if you keep having issues https://www.protondb.com/app/1343400.

As for Alt1, I've tried running it in Wine but the integrated browser doesn't work properly out of the box. Didn't look at it much more than that.

Your best bet is probably one of these native linux projects https://github.com/skillbert/alt1-electron https://github.com/whs/runekit. I didn't test either of these myself (I play OSRS) so your mileage may vary.
